packages/pdf-view/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@nativescript/pdf-view",
+  "version": "1.0.0",
+  "description": "A NativeScript plugin to display PDF files on iOS and Android",
+  "main": "pdf-view",
+  "typings": "pdf-view.d.ts",
```

Comment:
I would probably stick with `index.d.ts` and `index` for main - can just:
`pdf-view.android.ts` > `index.android.ts`
`pdf-view.ios.ts` > `index.ios.ts`
`pdf-view.common.ts` > `common.ts`
Then can just import the common imports is all. That follows more of a streamlined pattern 👍

Reply:
Renamed files and changed the module name to `pdf` as you suggested.